docs/user_workflow.adoc

```diff
@@ -0,0 +1,300 @@
+optimization scenarios. The first workflow, _instances consolidation in minimal compute
+nodes_ is demonstrated using the CLI. The second workflow, _workload stabilization_,
+is demonstrated the using `Horizon` UI and Prometheus metrics dashboards.
+
+== Requirements
+
+The example requires that the following setup is in place:
+
+* An OpenStack operators-based deployment with two or more Compute nodes
+* Nova Live migration is functional on your environment
+* Watcher has been deployed following the instructions in the User Installation Guide
+* Instances (Virtual Machines) have been created on the Compute nodes
+
+Test instances can be created using the `deploy-instance-demo.sh.template`:
+[source,bash]
+----
+include::_include/deploy-instance-demo.sh.template[]
+----
+and modifying the script for your environment as shown with the example commands below:
+
+[,console]
+----
+rm -f temp/deploy-instance-demo.sh
+cp deploy-instance-demo.sh.template temp/deploy-instance-demo.sh
+
+# $NETWORK_PREFIX is subnet prefixes.
+# This step is required if working within a shared tenant.
+# Each deployment must have their own prefix to avoid ip address collision
+sed -i "s/NETWORK_PREFIX/${NETWORK_PREFIX}/g" temp/deploy-instance-demo.sh
+
+# This example uses 8 instances.
+# Adjust the number per the number of instances to be used
+# in the test environment
+oc cp temp/deploy-instance-demo.sh openstackclient:/home/cloud-admin
+oc rsh openstackclient bash /home/cloud-admin/deploy-instance-demo.sh 8
+----
+
+You can check which hosts the VMs have been deployed on using the `--long` option
+on the `openstack server list` command:
+
+[,console]
+----
+$ oc rsh openstackclient openstack server list --project demo --long -c 'Name' -c 'Status' -c 'Host'
++--------+--------+-------------------------------+
+| Name   | Status | Host                          |
++--------+--------+-------------------------------+
+| test_7 | ACTIVE | compute1.ctlplane.localdomain |
+| test_6 | ACTIVE | compute2.ctlplane.localdomain |
+| test_5 | ACTIVE | compute1.ctlplane.localdomain |
+| test_4 | ACTIVE | compute2.ctlplane.localdomain |
+| test_3 | ACTIVE | compute1.ctlplane.localdomain |
+| test_2 | ACTIVE | compute2.ctlplane.localdomain |
+| test_1 | ACTIVE | compute1.ctlplane.localdomain |
+| test_0 | ACTIVE | compute1.ctlplane.localdomain |
++--------+--------+-------------------------------+
+----
+
+== Watcher Workflow (CLI)
+=== Optimization Scenario I: Instances Consolidation in Minimal Compute Nodes ===
+
+Information on the Watcher Strategies is available in the Openstack documentation.
+The server consolidation strategy is explained in the
+link:https://docs.openstack.org/watcher/2024.2/strategies/basic-server-consolidation.html[strategies documentation].
+
+.Procedure
+. Create the `Audit Template`
++
+[,console]
+----
+$ openstack optimize audittemplate create -s node_resource_consolidation \
```

Comment:
we need to add `oc rsh openstackclient` here and to the next commands, or instruct the user to run all the opesntack commands from the openstackclient pod